application/templates/pages/about/performance.md

```diff
@@ -0,0 +1,35 @@
+Performance indicators showing how the planning data platform is making it easier to find, use and trust planning and housing data in England.
+
+Last updated 19 July 2024.
+
+## Availability
+
+### 89 datasets on the platform
+
+**00 national datasets** available from **00 organisations**. 0 added last month (+/-0%).
+
+**00 datasets** provided by **00 local planning authorities**. 0 provided last month (+/-0%).
+
+0 new authorities providing data. +/-0% change from last week.
+
+## Quality
```

Comment:
Making an assumption in this section that we'll only report on the quality of data provided by LPAs. We won't report on the quality of national datasets provided by other organisations, e.g. public transport access nodes, as the organisations responsible for the data will be doing quality-checks.